index.html

```diff
@@ -892,11 +860,42 @@ <h4>Create Proof (eddsa-jcs-2022)</h4>
 is produced as output.
             </p>
 
-            <p>
-The algorithm is the same as the one described in Section
-[[[#create-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-            </p>
+            <ol class="algorithm">
+              <li>
+Set <var>options</var>.<var>@context</var> to
+<var>unsecuredDocument</var>.<var>@context</var>.
+              </li>
+              <li>
+Let |proof| be a clone of the proof options, |options|.
```

Comment:
I'm somewhat inclined to say nothing about this, but will mention a minor quibble here, which is that someone might complain / ask whether they can make a copy of options with this modification instead of modifying it directly. They can of course do this. Maybe others do not see this as a problem -- and if so, please ignore.

If it is a problem, one way to address this is to add something like `optionsCopy` (or set `options` to a copy of itself) and instead modify that (to add `@context`) and then make sure that is what is passed in all the subsequent steps. We could also perhaps just flip these steps and then modify the language where `options` is passed below in various steps to pass `proof` in its place as "options", if that makes sense.

Again, maybe this is a non-issue. We could perhaps leave a note that making a copy of `options` before modifying it as long as you pass the copy along is fine.

Reply:
I like the `optionsCopy` approach. Most JavaScript programmers have been burned by unintentional modifications to passed in objects and arrays to not like the "smell" of the current text. Good catch.